ci/run_pytest.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Copyright (c) 2025, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+# Support invoking run_pytest.sh outside the script directory
+cd "$(dirname "$(realpath "${BASH_SOURCE[0]}")")"/../dask_cuda
+
+rapids-logger "pytest dask-cuda"
+DASK_CUDA_TEST_SINGLE_GPU=1 \
+  DASK_CUDA_WAIT_WORKERS_MIN_TIMEOUT=20 \
+  UCXPY_IFNAME=eth0 \
+  UCX_WARN_UNUSED_ENV_VARS=n \
+  UCX_MEMTYPE_CACHE=n \
+  timeout 90m pytest \
```

Comment:
```suggestion
  timeout 90m python -m pytest \
```

Wheel test jobs are failing like this:

```text
timeout: failed to run command 'pytest': No such file or directory
```

([build link](https://github.com/rapidsai/dask-cuda/actions/runs/13133642104/job/36644321569?pr=1440))

Suspect you need `python -m pytest` here, as it was in `ci/test_wheel.sh`.